sample-apps/segmentation_spleen_postproc/lib/transforms.py

```diff
@@ -588,18 +622,30 @@ def __call__(self, data):
 #  Logits Save Transform
 ########################
 class WriteLogits(Transform):
-    def __init__(self, key, result="result"):
+    def __init__(self, key, ref_image=None, result="result", meta_key_postfix="meta_dict"):
         self.key = key
+        self.ref_image = ref_image if ref_image else key
         self.result = result
+        self.meta_key_postfix = meta_key_postfix
 
     def __call__(self, data):
+        logger = logging.getLogger(self.__class__.__name__)
+
         d = dict(data)
-        writer = Writer(label=self.key)
+        file_ext = "".join(pathlib.Path(d["image_path"]).suffixes)
```

Comment:
you can still use MONAI writer with nibabel=True.. and avoid writing 10 lines of code here.. 
`writer = Writer(label=self.key, nibabel=True)`

Reply:
Thanks! I will update this.